src/Microsoft.ML.StandardTrainers/Microsoft.ML.StandardTrainers.csproj

```diff
@@ -6,6 +6,10 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Research.SEALNet" Version="$(MicrosoftResearchSealPackageVersion)" />
```

Comment:
The new types in this assembly should be moved to the `Microsoft.ML.SEAL` assembly as well. We shouldn't add any new dependencies to the core Microsoft.ML package (which the StandardTrainers assembly is a part of).